xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsMacro.xml

```diff
@@ -375,11 +375,14 @@ require(['deferred!ckeditor', 'xwiki-suggestUsers', 'jquery', 'xwiki-meta'], fun
       <code>.xwiki-mention {
   background-color: $services.mentions.mentionsColor;
   border-radius: 10px;
-  padding: 2px 5px 2px 5px;
+  padding: 1px 5px 1px 5px;
+  border: 1px solid @dropdown-divider-bg;
 }
 
 .xwiki-mention.user.self {
   background-color: $services.mentions.selfMentionsColor;
+  color: $services.mentions.selfMentionsForeground;
```

Comment:
Looks like you forgot to add this as a new API in the script service.

Reply:
@surli 
Sorry for the delay,

I'm not very confident on script services yet. 
I added an API in https://github.com/xwiki/xwiki-platform/pull/2638/commits/e87afbf1a2d663125c07c90fa33eb19bf11ca0c4 
It definitely needs some review.

I added the unstable annotation onto this API since this is new.
From what I understand, the mention configuration source is registered under the same service name, we get the value of the mentionForeground color directly when using `$services.mentions.selfMentionsForeground`, but adding the `getSelfMentionsForeground` method onto the MentionScriptService allows us to:
* Properly make it an API
* Access it from other XWiki components if needed
* Access it from velocity script with `$services.mentions.getSelfMentionsForeground()`

Why did we not use this method for `selfMentionsColor` and we get the value directly from the configuration source with the variable name `$services.mentions.selfMentionsColor` ? Since we have a proper API getter I would have expected us to use it.

Let me know if I misunderstood something (which I probably did ^^`)